### PR TITLE
Add documentation regarding cyber security

### PR DESCRIPTION
**Pull request by freddidierRTE**
Nothing in release note

### CODE REVIEW

File: src/docs/asciidoc/security/index.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+== Code analysis 
+
+The code is analyzed at each commit by a static code analyzer (Sonar Cloud). The code is also regularly reviewed by the development team or by external security experts.
+
+Each modification is done via a pull request and is reviewed by at least one other developer.
+
+
+== Security Updates
+
+The OperatorFabric is regularly updated to integrate the latest security patches. To achieve this:
```
Comment (ClementBouvierN) [inline src/docs/asciidoc/security/index.adoc:23]: The OperatorFabric repository/code

File: src/docs/asciidoc/security/index.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+- The list of users is regularly reviewed.
+- A restricted number of users have the right to: 
+   - merge code in the main branch
+   - push docker images to the dockerhub registry.
+   - push libraries to mavencentral
+
+== Deployment Security
+
+The tool is to be used in a production environment, to do so the software shall be deployed by end user in a secure way.
```
Comment (ClementBouvierN) [inline src/docs/asciidoc/security/index.adoc:46]: should be deployed by the end user

File: src/docs/asciidoc/security/index.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+=== Inter-Services Communication
+
+The services within OperatorFabric communicate with each other using HTTP. This communication is not encrypted. The components should not be exposed to the internet and should be deployed in a secure network, either by:
+-  Using a single instance with docker-compose that does not expose the services to the outside world
+-  Using a Kubernetes cluster with a private network
+
+=== Access to card publication without authentication
+
+It is possible, by configuration, to bypass authentication for third-party tool card sending. This option is only there for legacy purposes and shall not be used in a production environment.
```
Comment (ClementBouvierN) [inline src/docs/asciidoc/security/index.adoc:70]: for legacy purposes and **should** not be used

File: src/docs/asciidoc/security/index.adoc (hunk 2 above)
Comment (freddidierRTE) [reply src/docs/asciidoc/security/index.adoc:46]: we keep shall as it is almost mandatory

File: src/docs/asciidoc/security/index.adoc (hunk 3 above)
Comment (freddidierRTE) [reply src/docs/asciidoc/security/index.adoc:70]: we keep shall as it is almost mandatory

**Pull request merged**